pages/bib/[id].tsx

```diff
@@ -68,7 +68,7 @@ export async function getServerSideProps({ params, resolvedUrl, req }) {
     id,
     bibParams
   )
-  const patronTokenResponse = await initializePatronTokenAuth(req)
+  const patronTokenResponse = await initializePatronTokenAuth(req.cookies)
```

Comment:
Not a part of this PR or for the Search Results page release but might as well update this now because we encounter this issue in the near future.